### ISSUE TITLE
Linux build is broken

### ISSUE THREAD

**Issue by 607011** [ola-ct/Qt-SESAM#36]
Lots of missing references to cryptopp lib :-(

**Comment by Wikinaut** [comment]
```
make[1]: Verzeichnis „/usr/local/src/Qt-SESAM/libSESAM“ wird betreten
g++ -c -pipe -O2 -w -fPIC -D_REENTRANT -DQT_WEBKIT -DCRYPTOPP_DISABLE_X86ASM -DCRYPTOPP_DISABLE_SSSE3 -DQTSESAM_VERSION=\"2.0.3\" -DQT_NO_DEBUG -DQT_CORE_LIB -DQT_SHARED -I../../../../share/qt4/mkspecs/default -I. -I../../../../include/QtCore -I../../../../include -I3rdparty/cryptopp -I3rdparty/bigint -I. -o util.o util.cpp

In file included from util.cpp:21:0:
util.h: In function ‘void SafeRenew(T&, T)’:
util.h:35:12: error: ‘nullptr’ was not declared in this scope
   if (a != nullptr)
            ^
util.h: In function ‘void SafeDelete(T&)’:
util.h:44:19: error: ‘nullptr’ was not declared in this scope
   SafeRenew<T>(a, nullptr);
                   ^
```

**Comment by 607011** [comment]
@Wikinaut Which version of g++ do you use? You need one that's able to compile c++11.

**Comment by Wikinaut** [comment]
OS openSUSE Tumbleweed

c++ --version
c++ (SUSE Linux) 4.8.3 20141208 [gcc-4_8-branch revision 218481]

**Comment by 607011** [comment]
@Wikinaut Hotfix: Replace all occurrences of `nullptr` with `NULL` or `0`.

Remedy is on the way …

**Comment by 607011** [comment]
@Wikinaut Strange … 

`nullptr` should be available in g++ since version 4.6. I'll check that.

**Comment by 607011** [comment]
Closed. Duplicate of #37
